README.md

```diff
@@ -167,29 +167,13 @@ class BookingEventConsumer : ProtoEventConsumer<BookingEvent.Builder>() {
 <table>
 <thead><tr><th>path</th><th>default value</th><th>explanation</th></tr></thead>
 <tr>
-<td><pre>tabour.retry-max-count</pre></td>
-<td><pre>1</pre></td>
-<td>Number of times that is gonna retry to publish  or consume an event</td>
-</tr>
-<tr>
-<tr>
-<td><pre>tabour.handler.thread-pool-size</pre></td>
+<td><pre>tabour.max-retry-count</pre></td>
 <td><pre>3</pre></td>
-<td>Size of the thread pool the the handler actors will use to consume the messages</td>
-</tr>
-<tr>
-<td><pre>tabour.handler.queue-size</pre></td>
-<td><pre>10</pre></td>
-<td>Size of the internal queue that the threads will use to pull from once a message is consumed</td>
-</tr>
-<tr>
-<td><pre>tabour.handler.thread-name-prefix</pre></td>
-<td><pre>""</pre></td>
-<td>Prefix of the threads that are created by the thread pool</td>
+<td>Number of times that is going to retry to publish  or consume an event</td>
 </tr>
 <tr>
 <td><pre>tabour.poller.poll-delay</pre></td>
-<td><pre>1 Second</pre></td>
+<td><pre>NA</pre></td>
 <td>Delay the poller should wait for the next poll after the previous poll has finished</td>
```

Comment:
What does NA mean here? There's no longer a poll delay?

Reply:
should mention that is not defined by default

Reply:
Used in a couple of places as Na, but yeah currently we don't have a poll delay and since it is configurable it can be set to be more